apps/web/lib/daily-webhook/triggerWebhooks.ts

```diff
@@ -0,0 +1,110 @@
+import type { TGetTranscriptAccessLink } from "@calcom/app-store/dailyvideo/zod";
+import getWebhooks from "@calcom/features/webhooks/lib/getWebhooks";
+import sendPayload from "@calcom/features/webhooks/lib/sendOrSchedulePayload";
+import getOrgIdFromMemberOrTeamId from "@calcom/lib/getOrgIdFromMemberOrTeamId";
+import logger from "@calcom/lib/logger";
+import { safeStringify } from "@calcom/lib/safeStringify";
+import { WebhookTriggerEvents } from "@calcom/prisma/enums";
+import type { CalendarEvent } from "@calcom/types/Calendar";
+
+const log = logger.getSubLogger({ prefix: ["daily-video-webhook-handler:triggerRecordingReadyWebhook"] });
+
+type Booking = {
+  userId: number | undefined;
+  eventTypeId: number | null;
+  eventTypeParentId: number | null | undefined;
+  teamId?: number | null;
+};
+
+const getWebhooksByEventTrigger = async (eventTrigger: WebhookTriggerEvents, booking: Booking) => {
+  const triggerForUser = !booking.teamId || (booking.teamId && booking.eventTypeParentId);
```

Comment:
nit: For clarity in understanding
```suggestion
  const triggerForUser = !isTeamBooking || isBookingForManagedEvent
```